www/docs/reactjs/introduction.mdx

```diff
@@ -161,6 +161,10 @@ export function App() {
 }
 ```
 
+:::note
+The usage of a useState for the creation of the query clients is there to prevent the client from being recreated on every render. This is exactly the same as using the `useMemo` hook.
+:::
+
```

Comment:
that's not quit correct as React preserves the right to forget memoization result from `useMemo` and call it again any time, which is not the case for `useState`. I'd say it's dangerous to use useMemo is such context.